src/python/doc/nerve_gic_complex_user.rst

```diff
@@ -9,21 +9,28 @@ Definition
 
 .. include:: nerve_gic_complex_sum.inc
 
-Visualizations of the simplicial complexes can be done with either
+We provide two types of classes for computing cover complexes: one is comprised of the :class:`~gudhi.sklearn.NerveComplex`, :class:`~gudhi.sklearn.GraphInducedComplex` and :class:`~gudhi.sklearn.MapperComplex` classes, which correspond to the Nerve, Graph Induced and Mapper complexes respectively, and are written in a scikit-learn format,
+while the other type is :class:`~gudhi.CoverComplex`, which only computes Nerve and Graph Induced complexes, but is a bit more flexible for input types (it can read inputs from paths to files for instance,
```

Comment:
Actually the `CoverComplex` class only computes nerves (with user-defined covers, so they can come from the preimages of a filter function, but you have to do it manually and provide the cluster assignments of each point directly) and GICs (you can ask for covers defined from preimages of a filter function, but it differs from mapper in the sense that connections between nodes are given by the underlying graph instead of non-empty intersection---this allows for, e.g., a partition of im(f) instead of a cover).

Reply:
Ah, ok, seeing both nerves and preimages of functions, I probably concluded too quickly that it must handle both together.